api/src/test/java/org/openmrs/layout/name/NameTemplateTest.java

```diff
@@ -20,18 +20,22 @@
 
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.openmrs.GlobalProperty;
 import org.openmrs.PersonName;
+import org.openmrs.api.context.Context;
 import org.openmrs.test.jupiter.BaseContextSensitiveTest;
+import org.openmrs.util.OpenmrsConstants;
 
 public class NameTemplateTest extends BaseContextSensitiveTest {
 	
+	protected static final String NAME_TEMPLATE_GP_DATASET_PATH = "src/test/resources/org/openmrs/include/nameSupportTestDataSet.xml";
```

Comment:
Is there any reason why this is protected instead of private?

Reply:
No...